guides/example/page_functionality_wix_sdk.md

```diff
@@ -16,33 +16,57 @@ You add permissions to your app in the [Developers Center](https://dev.wix.com/)
 
 After setting up permissions, our app can get a list of products from the site's store.
 
-We start by importing and installing the following:
+First, install the dependencies we'll need for working with the SDKs:
+
+```
+npm install @wix/sdk-react @wix/dashboard-react @wix/stores react-query
+```
+
+In our code, we start by wrapping our exported page component with the [`withDashboard`](TODO: link here) HOC. This will allow us to later use the [`useWixModules`](https://dev.wix.com/docs/sdk/api-reference/sdk-react/hooks#usewixmodules) hook to call Wix APIs.
```

Comment:
I think this section should be moved to the previous part of the tutorial where the query wrapper is set up. That way the build flows progressively and people don't have to go back and fix their code. @netanelgilad , if you agree, I can move it over after we deal with the other comments.

Reply:
I guess that makes sense,  the reason I didn't put it there is that the `withDashboard` doesn't have to do with page design, so didn't want to introduce it without explaining it. If you have a good way to make it "flow", go for it :)